codegen/end_to_end_test/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: end_to_end_test
```

Comment:
Please add this file to `.github/workflows/dart.yml`

Reply:
@klavs I've added a new section to workflows for `end_to_end_test` because it needs to run `built_runner` AND run tests.

Reply:
Do you know why the Activate Multipack built step is failing? It's breaking on `sortedTopologicalOrdering`, but I haven't dug much into multipack yet.